src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -320,17 +315,18 @@ public virtual bool Contains(Object item)
             if (item == null)
             {
                 for (int i = 0; i < _size; i++)
+                {
                     if (_items[i] == null)
                         return true;
+                }
                 return false;
             }
-            else
+            for (int i = 0; i < _size; i++)
```

Comment:
Thanks for understanding, I will assume you revert all the changes where you removed else statements.